include/deal.II/lac/petsc_ts.h

```diff
@@ -89,6 +90,7 @@ namespace PETScWrappers
       // Running parameters
       const std::string &options_prefix    = "",
       const std::string &ts_type           = "",
+      const std::string &problem_type      = "nonlinear",
```

Comment:
Adding the parameter anywhere else than the end, introduces some incompatibilities.